Makefile

```diff
@@ -122,19 +123,22 @@ normalized-link-check:
 .PHONY: markdown-link-check
 markdown-link-check: normalized-link-check
 	$(DOCKER_RUN) --rm \
+	    $(DOCKER_USER_IS_HOST_USER_ARG) \
 		--mount 'type=bind,source=$(PWD),target=/home/repo' \
-		lycheeverse/lychee \
+		$(LYCHEE_CONTAINER) \
 		--config home/repo/.lychee.toml \
 		--root-dir /home/repo \
 		--verbose \
+		--timeout=60 \
```

Comment:
unfortunately I'm still seeing sporadic timeouts, but let's keep this change anyways 👍